monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -382,6 +390,52 @@ class ConfigurePageComponent extends AuthComponent {
       }));
   }
 
+  sendLinuxMasqueStrings() {
```

Comment:
We'll need to add prefixes that allow the UI to tell the difference between strings and arbitrary bytes.